hatchet_sdk/clients/dispatcher/action_listener.py

```diff
@@ -90,27 +90,23 @@ def __post_init__(self):
             self.additional_metadata = {}
 
     @property
-    def otel_attributes(self) -> dict[str, Any]:
-        return flatten(
-            xs={
-                "worker_id": self.worker_id,
-                "tenant_id": self.tenant_id,
-                "workflow_run_id": self.workflow_run_id,
-                "get_group_key_run_id": self.get_group_key_run_id,
-                "job_id": self.job_id,
-                "job_name": self.job_name,
-                "job_run_id": self.job_run_id,
-                "step_id": self.step_id,
-                "step_run_id": self.step_run_id,
-                "retry_count": self.retry_count,
-                "child_workflow_index": self.child_workflow_index,
-                "child_workflow_key": self.child_workflow_key,
-                "parent_workflow_run_id": self.parent_workflow_run_id,
-                "action_payload": self.action_payload,
-            },
-            parent_key="",
-            separator=".",
-        )
+    def otel_attributes(self) -> dict[str, str | int]:
+        attrs: dict[str, str | int | None] = {
```

Comment:
missing 
`get_group_key_run_id`
`job_id`
`job_run_id`

intentional?

Reply:
ah good question, thanks for calling this out. do you think that the `job_id` and `job_run_id` are useful to people, or is that just an internal thing? `get_group_key_run_id` seems like maybe useful so can add that one!

Reply:
Likely group key is the only one